src/api/helpers/encrypt/generate-key/index.ts

```diff
@@ -0,0 +1,13 @@
+import crypto from 'crypto';
+import { FINANCIAL_DETAILS } from '../../../constants';
+
+const { KEY, ALGORITHM } = FINANCIAL_DETAILS.ENCRYPTION;
+
+/**
+ * generateKey
+ * generates key for encryption
+ * @returns {String} key
+ */
+const generateKey = () => crypto.createHash(ALGORITHM).update(KEY.SIGNATURE).digest('hex').substring(0, 32);
```

Comment:
Same as above RE `substring` and constants

can the `hex` come from constants?

Reply:
lint doesn't allow hex from constants here